.github/workflows/e2e_miscellaneous_tests.yml

```diff
@@ -63,6 +63,11 @@ jobs:
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-install --suite="featureGating"
         env: 
           VIDEO_RECORDING_IS_ENABLED: 0
+      - name: Run e2e Blog Dashboard Test
+        if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
+        run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-install --suite="blogDashboard"
```

Comment:
Are we expecting to have skip-build and prod-mode flag as well?

Reply:
@U8NWXD I am unable to run the tests with skip build locally, can it just be a issue with my system ?

Reply:
`--skip-build` works on CI because there we already built the Oppia app when we ran the previous test (featureGating in this case). That might not be the case on your local system

Reply:
![Screenshot from 2021-08-20 17-10-52](https://user-images.githubusercontent.com/68547101/130228011-3dd3809d-5a46-4dd2-a8e1-30fe976611a7.png)
With `--skip-build` this error comes on CI.

Reply:
No worries, mostly the change suggested by Sandeep should fix it.